ocaml/xapi/storage_migrate.ml

```diff
@@ -360,23 +371,32 @@ let with_activated_disk ~dbg ~sr ~vdi ~dp f =
   in
   finally
     (fun () ->
-      let path =
+      let path_and_nbd =
         Option.map
           (fun (vdi, backend) ->
-            let _, blockdevs, files, _ =
+            let _xendisks, blockdevs, files, nbds =
               Storage_interface.implementations_of_backend backend
             in
-            match (blockdevs, files) with
-            | {path} :: _, _ | _, {path} :: _ ->
+            match (files, blockdevs, nbds) with
```

Comment:
blockdevs and files got swapped, but the match remains the same, this doesn't look intentional